docs/docs/cloud/how-tos/copy_threads.md

```diff
@@ -0,0 +1,165 @@
+# Copying Threads
+
+You may wish to copy an existing thread in order to keep the existing history and experiment with different runs in the future.
+
+## Setup
+
+To copy a thread, we must first set up our client and also create a thread we wish to copy. 
```

Comment:
don't think we need to actually create one, we can just say "we assume you already have a thread to copy. if not, see this guide on how to make one"